plum/parametric.py

```diff
@@ -283,28 +283,48 @@ def __class_nonparametric__(cls):
         :mod:`plum.parametric` produces parametric subtypes of classes. This
         method can be used to get the non-parametric type of an object.
 
+        See Also
```

Comment:
This docstring is beautiful, but unfortunately I don't think it is rendered in the docs because it is a function inside a function. :(

Reply:
They should be exposed to the user on the derived classes, e.g. with `parametric(Class).__class_unparametrized__?` or `help(parametric(Class).__class_unparametrized__)` or `parametric(Class).__class_unparametrized__.__doc__`

Reply:
Yes, you're totally right! It's just a shame that it doesn't appear in the online docs.